test/contest/POC.t.sol

```diff
@@ -396,8 +396,8 @@ contract GovLstTest is UnitTestBase, PercentAssertions, TestHelpers, Eip712Helpe
     bytes memory _version,
     address _verifyingContract
   ) internal view returns (bytes32) {
-    bytes32 _seperator = _domainSeperator(_typeHash, _name, _version, _verifyingContract);
-    return keccak256(abi.encodePacked("\x19\x01", _seperator, _structHash));
+    bytes32 _separator = _domainSeperator(_typeHash, _name, _version, _verifyingContract);
+    return keccak256(abi.encodePacked("\x19\x01", _separator, _structHash));
```

Comment:
A couple more places where we made this typo.

```
test/GovLst.t.sol|399 col 33| bytes32 _separator = _domainSeperator(_typeHash, _name, _version, _verifyingContract);
test/helpers/Eip712Helper.sol|11 col 19| function _domainSeperator(bytes32 _typeHash, bytes memory _name, bytes memory _version, address _verifyingContract)
test/contest/POC.t.sol|399 col 33| bytes32 _separator = _domainSeperator(_typeHash, _name, _version, _verifyingContract);
test/WithdrawGate.t.sol|354 col 31| DOMAIN_SEPARATOR = _domainSeperator(EIP712_DOMAIN_TYPEHASH, "WithdrawGate", "1", address(withdrawGate));
```

Reply:
Fixed!